django/test/utils.py

```diff
@@ -850,3 +856,35 @@ def register_lookup(field, *lookups, lookup_name=None):
     finally:
         for lookup in lookups:
             field._unregister_lookup(lookup, lookup_name)
+
+
+class PlainPasswordHasher(BasePasswordHasher):
+    """
+    Plain password hashing algorithm (for tests use only).
+
+    Store password in plain text and use a static salt.
+    """
+    algorithm = 'plain'
+
+    def salt(self):
+        """Override base class method to return a fixed salt."""
+        return 'plain-salt'
```

Comment:
I've made it return a fixed value to otherwise we wouldn't have any `*` in the salt part of the assertion for the auth test:

https://github.com/django/django/blob/c492e40c1555f690103a16a0916baa7c1867d763/tests/auth_tests/test_views.py#L1251-L1257